packages/esm-patient-common-lib/src/orders/useOrders.ts

```diff
@@ -50,3 +58,15 @@ export function useOrderTypes() {
     isValidating,
   };
 }
+
+export function getDrugOrderByUuid(orderUuid: string) {
+  const customRepresentation =
```

Comment:
Be nice to move this outside the function, since it's just a constant.

Reply:
Done